src/main/java/io/github/queerbric/queerbrics/mixin/ItemMixin.java

```diff
@@ -0,0 +1,43 @@
+package io.github.queerbric.queerbrics.mixin;
+
+import java.util.Random;
+
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import io.github.queerbric.queerbrics.datatypes.entity.ThrowableBrickEntity;
```

Comment:
alphabetic order?

Reply:
they are alphabetical within groups, not sure how it's determined but it's consistent with the rest of the files